.github/workflows/mgrpush-unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run unit tests for mgr-push
+
+on:
+  pull_request:
+    paths: 'client/tools/mgr-push/*'
+
+jobs:
+  mgrpush_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
How did you come to the decision to use Python 3.8?

Reply:
The GitHub Action was failing running `pytest` with Python 3.11 because `python/uyuni/common/checkum.py` uses `inspect.getargspec()` and this function has been [removed](https://docs.python.org/3.11/library/inspect.html) in Python 3.11. However, it it still there for Python 3.10 (although marked as deprecated).

Reply:
I think it would be good to test with the versions that will be used in production. With GH we don't pay for additional Python version, I would test with all of the supported ones. I created a [wiki page](https://github.com/uyuni-project/uyuni/wiki/Python-Version-Overview) with an overview

Reply:
I'm having trouble testing with Python 2.7 as `httplib` doesn't seem to be available in the runner, neither available via `pip`.

Reply:
Regarding the 3.x versions, 3.4 is available for Ubuntu [18.04](https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json) and this runner image has been marked as [deprecated](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/).